services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -271,3 +280,37 @@ async def service_restart_containers(
         raise LegacyServiceIsNotSupportedError() from error
 
     return NoContentResponse()
+
+
+@router.post(
+    "/{project_id}/project-networks:update",
```

Comment:
In general I agree, since *Update* is already part of the standard methods. Please read https://google.aip.dev/134

Here IMO there is a fundamental problem in the way the resources in this API are designed though .. SEE https://google.aip.dev/121.  For that reason this "update" might not fit the purpose with a PATCH. In summary, if at the begining the resource does not follow the established rules ... then the remaining recommendations do not fit that nicely anymore